deny.toml

```diff
@@ -0,0 +1,239 @@
+[licenses]
+# The default list of licenses is empty, so we specify what we currently use
+# while open to reasonable extensions to this list.
```

Comment:
"while being open"?

Reply:
IMHO a more passive voice reads a bit better. Do you insist?

Reply:
Nope, I just thought it's a typo.